source/_integrations/slow_pwm.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: Slow PWM
+description: Convert one or multiple switches to a numeric input using slow pulse width modulation
+ha_category:
+  - Number
+ha_release: '2023.05'
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@antonverburg'
+ha_domain: slow_pwm
+ha_platforms:
+  - number
+ha_integration_type: integration
+---
+
+The `slow_pwm` platform is a `number`. It can be used to control one or multiple digital switches in an analog control algorithm. 
+The switches will be controlled by pulse width modulation (PWM, see https://en.wikipedia.org/wiki/Pulse-width_modulation). 
+This enables the usage of digital switches in modulated modes. This is typically useful in slow systems like floor heater or boiler heater controller systems. The [PID controller](/integrations/pid_controller) and the [PID thermostat](/integrations/pid_thermostat)  can use this number as regulated output. 
+
+## Configuration
```

Comment:
This header is already provided by the include below, did you test this before opening the PR?